mlflow/R/mlflow/R/model.R

```diff
@@ -3,23 +3,20 @@
 #' Saves model in MLflow format that can later be used
 #' for prediction and serving.
 #'
-#' @param x The serving function or model that will perform a prediction.
+#' @param model The model that will perform a prediction.
 #' @param path Destination path where this MLflow compatible model
 #'   will be saved.
-#' @param r_dependencies Optional vector of paths to dependency files
-#'   to include in the model, as in \code{r-dependencies.txt}
-#'   or \code{conda.yaml}.
 #' @param conda_env Path to Conda dependencies file.
 #'
 #' @importFrom yaml write_yaml
 #' @export
-mlflow_save_model <- function(x, path = "model", r_dependencies=NULL, conda_env=NULL) {
+mlflow_save_model <- function(model, path = "model", conda_env=NULL) {
```

Comment:
We can probably remove the `conda_env` here and also from the signature of `mlflow_save_flavor()` since not all flavors care about conda. We can use `...` and forward it to the `mlflow_save_flavor()` call.